packages/browser/test/unit/tracekit/misc.test.ts

```diff
@@ -26,4 +26,107 @@ describe('Tracekit - Misc Tests', () => {
       },
     });
   });
+
+  it.only('should parse SecurityError', () => {
```

Comment:
```suggestion
  it('should parse SecurityError', () => {
```

I guess this was forgotten ;)